Add hints for transit build. (#154)
gis-ops/docker-valhalla: README.md

- `build_time_zones`: `True` builds the timezone db which is needed for time-dependent routing. `Force` will do the same, but first delete the existing db. Default `False`.
- `build_transit`: `True` will attempt to build transit tiles if none exist yet. `Force` will remove existing transit **and** routing tiles. Default `False`.
- `build_tar` (since 29.10.2021/v`3.1.5`): `True` creates a tarball of the tiles including an index which allows for extremely faster graph loading after reboots. `Force` will do the same, but first delete the existing tarball. Default `True`.

> - `server_threads`: How many threads `valhalla_build_tiles` will use and `valhalla_service` will run with. Default is the value of `nproc`. If valhalla kills it self when building transit, lower this number. 

Reviewer: This is a general advice IMO, also without transit it’ll be killed when building the graph. I’d suggest to remove „transit“

Author: Good point. I added this in another commit to "building tiles" instead of "building transit"

Valhalla supports reading raw GTFS feeds to build transit into its graph, see the [docs](https://valhalla.github.io/valhalla/api/turn-by-turn/api-reference/#sample-json-payloads-for-multimodal-requests-with-transit) for more details.

> Put the gtfs fees in dedicated subfolders in your gtfs folder, e.g. `gtfs_feeds/berlin/`, otherwise the files will not be found.

Reviewer: ```suggestion
Put the unzipped GTFS feeds as subfolders in the main gtfs folder, e.g. `gtfs_feeds/berlin/`, otherwise the files will not be found.
```
